spec/comm.mediawiki

```diff
@@ -182,14 +182,24 @@ connection is established, the client will supply their account ID and signature
 
 If a client unexpectedly disconnects with active orders, the orders may match in
 the client's absence. A list of any pending matches is included in the response.
+If the client has a broken rule, they will not be able to trade for the duration
+of their penalization.
 
 <code>result</code>
 {|
 ! field   !! type   !! description
 |-
-| matches || &#91;object&#93; || list of [[orders.mediawiki/#Match_negotiation|Match objects]]
+| matches          || &#91;object&#93; || list of [[orders.mediawiki/#Match_negotiation|Match objects]]
 |-
-| sig     || string || hex-encoded server's signature of the serialized connection data
+| broken rule      || int || the client's last broken [[community.mediawiki/#Rules_of_Community_Conduct|rule]]. omitted if in good standing
+|-
+| penalty time     || int || the UNIX time in milliseconds at which the penalization occured. omitted if in good standing
+|-
+| penalty duration || int || the duration in milliseconds of the penalization. omitted if in good standing
+|-
+| penalty details  || string || a message describing the penalization. omitted if in good standing
```

Comment:
We'll have to get more sophisticated eventually, but this seems like a reasonable approach to get the information into the clients hands for now. Have you considered an`object` field.
```
"penalty": {
    "rule": ...,
    "time": ...,
    ...
}
```

The same object could be used in the `penalize` notification, and would be defined in ***community.mediawiki***, and only referenced here in the connect response fields.

Reply:
Good idea!